<launch>
  <arg name="joy_dev" default="/dev/input/js0"/>

  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.12" />
  </node>

  <!-- Axes -->
  <param name="rate" value="5" type="int"/>
  <param name="axis_linear_x" value="0" type="int"/>
  <param name="axis_linear_y" value="1" type="int"/>
  <param name="axis_linear_z" value="4" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="scale_angular" value="1" type="double"/>
  <node pkg="svenzva_joy" type="svenzva_joystick_controller" name="svenzva_joy"/>

</launch>
